src/dbnode/storage/series/buffer.go

```diff
@@ -60,6 +60,10 @@ const (
 	// enabled to see if this is a sane number.
 	evictedTimesArraySize = 8
 	writableBucketVer     = 0
+
+	// NB: this is the value all points will be truncated to.
+	truncatedValue float64 = 0
+	truncatedUnit          = xtime.Second
```

Comment:
We also probably don't need to override the unit no?  Client should always be sending the same unit unless it has a bug.

Reply:
Yeah fair, this was mostly to kinda standardize all values going into the index but don't suppose it matters very much here